### DISCUSSION TITLE
Fix the monitor change buttons on the toolbar .. and not only this

### DISCUSSION THREAD

**Discussion by gh-Destiny** [rustdesk/rustdesk#7428] [General]
In the current implementation of the toolbar, the buttons for precisely selecting from two or more monitors located on a connected remote work machine **are hidden**. Yes, I see it exactly like that.  Not less... And I can't even imagine in a nightmare for what purpose they are hidden, and by default I have to open the click-menu on the 'monitor change' button **every time** and only THEN allow myself to select the exact monitor which I need right now.
No, no, and no again. These selection buttons should be on the same **first level** as all the current buttons on the toolbar!
Choosing a monitor should be **an instant task** and be solved instantly and intuitively.
Actually, this is my first and **most important** suggestion.

BUT I will say even more than that - it should be possible to fix the ENTIRE toolbar within the working area of the main window!
EVEN IF this leads to an unambiguous reduction in the workspace, where the entire image from the remote machine is displayed!
And thus allow yourself to choose which buttons/actions/commands will be fixed there.

**Comment by gh-Destiny** [comment]
Rather strange that nobody from developers do not notice/leave some input for such simple FR.
Does this section exist only for the NON-developers talking?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/3670

**Comment by gh-Destiny** [comment]
This solution corrects only one drawback - I can see the monitor buttons right away on the taskbar.
But I still don’t see the possibility of fixing the ENTIRE task panel within the working area of ​​the rustdesk app-window.
Those. I always have to either hide or pull out this task panel so that I can switch freely between monitors,
or hide the panel so that it does not obscure the desired part of the top-zone on the remote computer.

It would be better if I IMMEDIATELY integrate the ENTIRE taskbar into the workspace of the remote machine control screen.
In other words, it would be better to repeat how it was done in anydesk, for example. It's really convenient and doesn't make me constantly touch this taskbar, because I always see EVERYTHING RIGHT AWAY. And the buttons on the taskbar AND THE ENTIRE work area of the remote machine.
